app/services/error-handler.js

```diff
@@ -0,0 +1,27 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['@ember/service' import is duplicated.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011987)